caput/time.py

```diff
@@ -1075,11 +1077,22 @@ def timescale(self):
         try:
             self._timescale = self.load.timescale(builtin=False)
             return self._timescale
-        except IOError as e:
-            raise IOError(
-                "Could not find existing Skyfield timescale data at %s"
-                % self.load.directory
-            ) from e
+        except IOError:
+            warnings.warn(
+                "Could not get timescale data from an official source. Trying the "
+                "CHIME mirror, but the products are likely out of date."
+            )
+            try:
+                self.load.download(self.mirror_url + "Leap_Second.dat")
+                self.load.download(self.mirror_url + "finals2000A.all")
+
+                self._timescale = self.load.timescale(builtin=False)
+            except IOError as e:
+                raise IOError(
+                    "Could not get timescale either from an existing installation at "
+                    f"{self.load.directory}, an official download source or the CHIME "
+                    "mirror. Good luck."
+                ) from e
```

Comment:
Add a more helpful error message suggesting the user download themselves? Something like "If you have a functioning mirror, try using ctime.skyfield_wrapper.load to manually download the data."

Reply:
Yeah, I can do that.